<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScaleCalcWidget</class>
 <widget class="QWidget" name="ScaleCalcWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>680</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_Mode">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>80</y>
     <width>341</width>
     <height>421</height>
    </rect>
   </property>
   <property name="title">
    <string>Scale参数模式</string>
   </property>
   <widget class="QRadioButton" name="radioButton_Comm_Scale">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>231</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>使用指定校准参数</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Ref_Result">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>231</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>使用GRst里面动态参数</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_DefaultValue">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>301</width>
      <height>171</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="title">
     <string>默认错误数据</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_24">
         <property name="text">
          <string>像素长度:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtPixLen">
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_25">
         <property name="text">
          <string>物理长度:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtPhyLen">
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>281</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>动态标定参数索引:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtScalePara">
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_Ref_Point">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>520</y>
     <width>301</width>
     <height>151</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="title">
    <string>引用距离任务</string>
   </property>
   <widget class="QComboBox" name="comboBoxPoint">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>距离任务：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>距离结果:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxPointResult">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
